CTCommons/MSTranscription/MSTranscriptionService.cs

```diff
@@ -39,164 +40,184 @@ public async Task<MSTResult> RecognitionWithVideoStreamAsync(FileRecord videoFil
+                                {
+                                    _logger.LogInformation($"CANCELED: ErrorCode={e.ErrorCode.ToString()} Reason={e.Reason}");
+                                    _slackLogger.PostErrorAsync(new Exception($"Transcription Failure"),
+                                        $"Transcription Failure").GetAwaiter().GetResult();
                                 }
-
-                                _logger.LogInformation($"Retrying, LastSuccessTime={lastTime.ToString()}");
-                                lastSuccessfulTime = lastTime;
-                            } else if (e.ErrorCode != CancellationErrorCode.NoError)
-                            {
-                                _logger.LogInformation($"CANCELED: ErrorCode={e.ErrorCode.ToString()} Reason={e.Reason}");
-                                _slackLogger.PostErrorAsync(new Exception($"Transcription Failure"),
-                                    $"Transcription Failure").GetAwaiter().GetResult();
                             }
-                        }
 
-                        stopRecognition.TrySetResult(0);
-                    };
+                            stopRecognition.TrySetResult(0);
+                        };
 
-                    recognizer.SessionStarted += (s, e) =>
-                    {
-                        _logger.LogInformation("\nSession started event.");
-                    };
+                        recognizer.SessionStarted += (s, e) =>
+                        {
+                            _logger.LogInformation("\nSession started event.");
+                        };
 
-                    recognizer.SessionStopped += (s, e) =>
-                    {
-                        _logger.LogInformation("\nSession stopped event.");
-                        _logger.LogInformation("\nStop recognition.");
-                        stopRecognition.TrySetResult(0);
-                    };
-
-                    // Starts continuous recognition. Uses StopContinuousRecognitionAsync() to stop recognition.
-                    await recognizer.StartContinuousRecognitionAsync().ConfigureAwait(false);
-
-                    // Waits for completion.
-                    // Use Task.WaitAny to keep the task rooted.
-                    Task.WaitAny(new[] { stopRecognition.Task });
-
-                    // Stops recognition.
-                    await recognizer.StopContinuousRecognitionAsync().ConfigureAwait(false);
-
-                    return new MSTResult {
-                        Captions = captions,
-                        ErrorCode = errorCode,
-                        LastSuccessTime = lastSuccessfulTime
-                    };
+                        recognizer.SessionStopped += (s, e) =>
+                        {
+                            _logger.LogInformation("\nSession stopped event.");
+                            _logger.LogInformation("\nStop recognition.");
```

Comment:
Can this be combined into one line? `_logger.LogInformation("\nSession stopped event.\nStop recognition.");`

Reply:
I'll fix this and also add the videoID to the log messages

Reply:
Resolved in https://github.com/classtranscribe/WebAPI/pull/101/commits/8773f6f0aa7747196d8d3ed4a53a097372dab85c